packages/charts/src/components/ScatterChart/ScatterChart.tsx

```diff
@@ -0,0 +1,242 @@
+import { YAxisTicks } from '@ui5/webcomponents-react-charts/lib/components/YAxisTicks';
+import { LineChartPlaceholder } from '@ui5/webcomponents-react-charts/lib/LineChartPlaceholder';
+import { useLegendItemClick } from '@ui5/webcomponents-react-charts/lib/useLegendItemClick';
+import React, { CSSProperties, FC, forwardRef, Ref, useCallback, useMemo } from 'react';
+import {
+  CartesianGrid,
+  Legend,
+  Scatter,
+  ScatterChart as ScatterChartLib,
+  ReferenceLine,
+  Tooltip,
+  XAxis,
+  YAxis,
+  ZAxis,
+  Label
+} from 'recharts';
+import { useChartMargin } from '../../hooks/useChartMargin';
+import { useLongestYAxisLabel } from '../../hooks/useLongestYAxisLabel';
+import { usePrepareDimensionsAndMeasures } from '../../hooks/usePrepareDimensionsAndMeasures';
+import { useTooltipFormatter } from '../../hooks/useTooltipFormatter';
+import { IChartBaseProps } from '../../interfaces/IChartBaseProps';
+import { IChartMeasure } from '../../interfaces/IChartMeasure';
+import { defaultFormatter } from '../../internal/defaults';
+import { tickLineConfig, tooltipContentStyle, tooltipFillOpacity, xAxisPadding } from '../../internal/staticProps';
+import { useObserveXAxisHeights } from '../../hooks/useObserveXAxisHeights';
+
+interface MeasureConfig extends IChartMeasure {
+  /**
+   * Line Width
+   * @default 1
+   */
+  width?: number;
+  /**
+   * Line Opacity
+   * @default 1
+   */
+  opacity?: number;
+  /**
+   * Defines axis of measures
+   * @default 1
+   */
+  axis: 'x' | 'y' | 'z';
+}
+
+interface ScatterDataObject {
+  label?: string;
+  data?: any[];
+  color?: CSSProperties['color'];
+}
+
+export interface ScatterChartProps extends IChartBaseProps {
+  dataset?: ScatterDataObject[];
+  /**
+   * An array of config objects. Each object is defining one line in the chart.
+   *
+   * <h4>Required properties</h4>
+   * - `accessor`: string containing the path to the dataset key this line should display. Supports object structures by using <code>'parent.child'</code>.
+   *   Can also be a getter.
+   *
+   * <h4>Optional properties</h4>
+   *
+   * - `label`: Label to display in legends or tooltips. Falls back to the <code>accessor</code> if not present.
+   * - `color`: any valid CSS Color or CSS Variable. Defaults to the `sapChart_Ordinal` colors
+   * - `formatter`: function will be called for each data label and allows you to format it according to your needs
+   * - `hideDataLabel`: flag whether the data labels should be hidden in the chart for this line.
+   * - `DataLabel`: a custom component to be used for the data label
+   * - `width`: line width, defaults to `1`
+   * - `opacity`: line opacity, defaults to `1`
+   *
+   */
+  measures?: MeasureConfig[];
+}
+
+const measureDefaults = {
+  formatter: defaultFormatter
+};
+
+/**
+ * <code>import { LineChart } from '@ui5/webcomponents-react-charts/lib/LineChart';</code>
+ */
+const ScatterChart: FC<ScatterChartProps> = forwardRef((props: ScatterChartProps, ref: Ref<any>) => {
```

Comment:
I don't like the `Ref<any>`.. I think it should be something like `Ref<HTMLDivElement>` or whatever the most outer element is.